src/intTest/java/src/ScreenshotTakerTest.java

```diff
@@ -0,0 +1,70 @@
+package src;
+
+import org.junit.Test;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+import static junit.framework.TestCase.assertTrue;
+import static junit.framework.TestCase.fail;
```

Comment:
>import org.junit.Test;

Is not used.

>import static junit.framework.TestCase.assertTrue;
>import static junit.framework.TestCase.fail;

From whole JUnit you are using only methods to make unchecked exceptions. `assertTrue` is equal to `if (condition) throw AssertException(some logic to build clear message)`.

1. They can be implemented manually, not need to to ask JUnit dependency only for that.
1. You are interested only in process exit code, such exceptions can be replaced with printing error details and `System.exit(1)`.